src/solace_ai_connector/components/general/db/mongo/mongo_search.py

```diff
@@ -0,0 +1,97 @@
+"""MongoDB Agent Component for handling database search."""
+import json
+
+from .mongo_handler import MongoHandler
+from ....component_base import ComponentBase
+
+
+info = {
```

Comment:
[SUG]: move the info and __init__ to a mongo_base class and inherit the class in the mongo_insert and search components.

Reply:
Updated to use a base class